packages/api/lib/config.ts

```diff
@@ -2,19 +2,21 @@ const config = require('@iteam/config')({
   file: `${__dirname}/../config.json`,
   defaults: {
     PORT: 4000,
-    DRONE_URL: 'http://localhost:3030',
-    OSRM_URL: 'http://localhost:5000',
-    ELASTIC_URL: 'http://localhost:9200',
-    WEBHOOK_URL: 'http://localhost:4000/status',
+    DRONE_URL: process.env.DRONE_URL || 'http://localhost:3030',
```

Comment:
Are we sure we need to explicitly override these using `process.env.XYZ`? 

From what I can tell `environment`-variables should override defaults: https://iteam1337.github.io/#/config/examples

So, think we can safely leave them out!

Reply:
Ah ok! Nice find